package.json

```diff
@@ -38,27 +38,33 @@
   },
   "homepage": "https://github.com/jantimon/favicons-webpack-plugin",
   "devDependencies": {
-    "ava": "^1.4.1",
+    "ava": "^2.2.0",
     "dir-compare": "^1.7.2",
-    "fs-extra": "^8.0.1",
-    "html-webpack-plugin": "^3.2.0",
+    "fs-extra": "^8.1.0",
+    "html-webpack-plugin": "^4.0.0-beta.5",
     "nyc": "^14.1.1",
     "standard-version": "7.0.0",
-    "webpack": "^4.35.0",
+    "webpack": "^4.35.2",
     "webpack-cli": "^3.3.5",
     "webpack-dev-server": "^3.7.2",
     "webpack-merge": "^4.2.1"
   },
   "dependencies": {
-    "cache-loader": "^3.0.1",
-    "camelcase": "^5.3.1",
+    "cache-loader": "^4.1.0",
     "favicons": "5.4.1",
-    "find-cache-dir": "^2.1.0",
+    "find-cache-dir": "^3.0.0",
     "find-root": "^1.1.0",
     "loader-utils": "^1.2.3",
-    "parse-author": "^2.0.0"
+    "parse-author": "^2.0.0",
+    "parse5": "^5.1.0",
```

Comment:
I would like to avoid adding such a big dependency - however I don't have a better idea right now.  
Is there any other possibility to parse the tags coming from favicons?

Reply:
How do you measure how "big" a dependency is?

I have no strong opinion on which alternative to use, as long as we don't manually implement it. Do you have anything in mind?

Reply:
Adding many dependencies which usually do a lot more than we need creates always the risk that the small fraction we require will break someday and we also have to make sure that this package doesn't introduce security vulnerabilities. Every packages you add adds also maintenance work. 

But here I guess parse5 seems to be the best choice 👍 - it comes without any sub dependencies and there is no built in node way to deal with xml entities. **=> Lets use `parse5`**